common/app/common/commercial/hosted/hardcoded/Formula1HostedPages.scala

```diff
@@ -0,0 +1,46 @@
+package common.commercial.hosted.hardcoded
+
+import common.commercial.hosted._
+import conf.Configuration.site.host
+import conf.Static
+import conf.switches.Switches
+
+object Formula1HostedPages {
+
+  private val campaign = HostedCampaign(
+    id = "singapore-grand-prix",
+    name = "Singapore Grand Prix",
+    owner = "First Stop Singapore",
+    logo = HostedLogo("https://static.theguardian.com/commercial/hosted/formula1-singapore/2016_TrackLogo_RGB_FullColour.png"),
+    cssClass = "f1-singapore"
+  )
+
+  private val cta = HostedCallToAction(
+    url = "https://www.firststopsingapore.com/singapore-airlines-singapore-grand-prix-packages/?utm_source=guardian&utm_medium=editorial&utm_campaign=singaporegrandprix",
+    image = "https://static.theguardian.com/commercial/hosted/formula1-singapore/SGPGuardianImage.jpg",
+    label = "Discover the 2016 Singapore Grand Prix",
+    trackingCode = "singapore-grand-prix",
+    btnText = "Book now"
+  )
+
+  val overviewPageName = "overview"
+
+  val overviewArticlePage = HostedArticlePage2(
+    campaign,
+    pageUrl = s"$host/advertiser-content/${campaign.id}/$overviewPageName",
+    pageName = overviewPageName,
+    title = "Overview",
+    standfirst = "Singapore has been the proud host of the world's only Formula One night race since 2008. The backdrop of heritage buildings, modern architecture and breathtaking skyline culminates in an unparalleled combination of sights, sounds and a festive atmosphere.",
+    facebookImageUrl = Static("images/commercial/zootropolis.png"),
```

Comment:
Be careful with these static references, as that's what caused Code to stop working last time.
You can check that it will work by setting `assets.useHashedBundles=true` in frontend properties.